client/components/utils/cursors-plugin.js

```diff
@@ -0,0 +1,86 @@
+import {_} from 'meteor/underscore';
+import {Plugin} from 'prosemirror-state';
+import {Decoration, DecorationSet} from 'prosemirror-view';
+
+/**
+ * Helper function that creates a DOM.node with the cursor to render.
+ * @param {*} color - String with the hex color to use.
+ */
+function createCaret(color, name, avatar) {
```

Comment:
This function would be more readable with empty lines splitting sections of the code.